client/src/components/FilterForm.vue

```diff
@@ -53,38 +49,57 @@ export default {
   name: 'FilterForm',
   title: 'Filter',
   id: 'filter',
+  model: {
+    prop: 'filterData',
+    event: 'updateFilter',
+  },
   props: {
-    deltaC: Number,
-    setDeltaC: Function,
-    filterModel: String,
-    setFilterModel: Function,
-    filterParams: Object,
-    setFilterParams: Function,
+    filterData: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      allFilterData: BACKEND_CONSTANTS.Filter_params,
+      filterChoices,
+      filterChoice: BACKEND_CONSTANTS.filter_model,
+    };
   },
-  data: () => ({
-    filterChoices,
-    filterChoice: '',
-  }),
   components: {
-    InputField,
+    DoubleField,
   },
   methods: {
-    createSetCurrentValueFunc(filterType, varName) {
+    createSetParamValueFunc(filterType, varName) {
       return (newValue) => {
         // Set the new value temporarily
-        this.filterParams[filterType][varName] = newValue;
+        this.filterData.filter_params[varName] = newValue;
+        this.allFilterData[filterType][varName] = newValue;
```

Comment:
Same feedback as before. Don't write to props!

Reply:
Same comment as in the other PR then looking for feedback on how to do this instead.

Reply:
Done